data/core/units/elves/Hero.cfg

```diff
@@ -15,7 +15,7 @@
     advances_to=Elvish Champion
     cost=30
     usage=fighter
-    description= _ "Elves, though not warlike by nature, are swift to learn; a relatively small amount of experience will turn an elf from a competent fighter into a master of combat. Those who are honored as heroes are strong with both sword and bow, and their skill once acquired does not fade for lack of practice."
+    description= _ "Elves that have grown more adept at swordsmanship often forego a shield in favor of a secondary shortsword, useful for both parrying enemy attacks and sneaking in extra strikes when engaging in melee combat. These diminutive blades are often known as 'Västaja', little heroes, for their third use as replacement swords for weaponless allies. More inexperienced fighters are easily disarmed by stronger opponents and quickly overwhelmed when only left with a bow to fight with. By offering their parrying swords to their more vulnerable comrades, elvish heroes grant them a second chance to fight or to retreat with their lives."
```

Comment:
What is "Västaja" from? It doesn't sound very elvish to me, to be honest; if you're going to toss in some samples of the Elvish language, you should probably make them sound similar in structure to the Elvish names.

Reply:
I made it up, but I was actually hoping someone (maybe you?? :p) could suggest a better name. I'd still like to name the blades in the description, since it'll add another element to the lore.